package.json

```diff
@@ -29,8 +29,8 @@
     "eslint": "^8.52.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-sonarjs": "^0.22.0",
-    "typescript": "^5.0.4",
+    "typescript": "5.2.2",
```

Comment:
Is this typescript update required for this issue?

Reply:
I had to make this change, because with the previous configuration Typescript 5.3 got installed and ESLint started to complain about incompatibility. By fixing the version to the last version before 5.3 this issue is resolved.